src/igvfd/tests/test_types_whole_organism.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+
+def test_summary(testapp, whole_organism, rodent_donor, parent_rodent_donor_1, human_donor):
+    res = testapp.get(whole_organism['@id'])
+    assert res.json.get('summary') == 'whole organism, male Mus musculus strain1'
+    testapp.patch_json(
+        whole_organism['@id'],
+        {
+            'lower_bound_age': 1,
+            'upper_bound_age': 1,
+            'age_units': 'hour'
+        }
+    )
+    res = testapp.get(whole_organism['@id'])
+    assert res.json.get('summary') == 'whole organism, male Mus musculus strain1 (1 hour)'
+    testapp.patch_json(
+        whole_organism['@id'],
+        {
+            'donors': [rodent_donor['@id'], parent_rodent_donor_1['@id']],
+            'lower_bound_age': 1,
+            'upper_bound_age': 5,
+            'age_units': 'hour'
+        }
+    )
+    res = testapp.get(whole_organism['@id'])
+    assert res.json.get('summary') == '2 whole organisms, mixed sex Mus musculus strain1, strain2 (1-5 hours)'
+    testapp.patch_json(
+        human_donor['@id'],
+        {
+            'sex': 'male'
+        }
+    )
+    testapp.patch_json(
+        whole_organism['@id'],
+        {
+            'donors': [human_donor['@id'], rodent_donor['@id']],
+            'lower_bound_age': 10,
+            'upper_bound_age': 10,
+            'age_units': 'month'
+        }
+    )
+    print(res.json.get('summary'))
+    res = testapp.get(whole_organism['@id'])
+    assert res.json.get('summary') == '2 whole organisms, males (10 months)'
```

Comment:
i think rodent/human should say mixed taxa?

Reply:
If this is desired why doesn't mixed taxa return anything in the calculated property for taxa?

Reply:
it is not calculated, because it is mixed. the summary however could "tap" into donors to receive the taxa.  I am trying to say that strict property listing taxa - should avoid saying "mixed" as it is not a taxa. the summary which is human human-readable string can say whatever makes sense. in this case maybe you can say "2 whole organisms, homo sapiens and mus musculus, males (10 months)" or something along these lines. it seems wrong not to say the taxa in the summary if they are available in donors.

Reply:
Ok. I'll change this.